README.md

```diff
@@ -9,13 +9,19 @@ The DURI ([GA4GH::DURI](http://ga4gh-duri.github.io)) Work Stream is one of the
 
 The GA4GH DURI workstream is led by [Jaime Guidry Auvil](https://datascience.cancer.gov/content/jaime-m-guidry-auvil-phd) and [Tommi Nyronen](https://www.linkedin.com/in/nyronen/). All members of this team must follow the [GA4GH Standards of Professional Conduct](https://www.ga4gh.org/wp-content/uploads/GA4GH-Standards-for-Professional-Conduct_22-Jan-2018.pdf).
 
+This team has two main standards:
```

Comment:
The list does not show up as numbers nor bullets. Perhaps add "-  " to each line below?

Reply:
Fixed!